apps/fhir/bluebutton/tests/fhir_resources/eob_read_v2.json

```diff
@@ -0,0 +1 @@
+{"detail": "The requested resource does not exist"}
```

Comment:
This looks like an error?

Reply:
I ran into that once but on the subsequent local runs, it never occurred again...

Reply:
Right, so why is it included here?  I don't see it being used in testing as far as I can see, so I would just delete it.

Reply:
OH, I see what you are referring to, eob_read_v1.json and eob_read_v2.json are no longer used in unittests, will remove